src/ca65/main.c

```diff
@@ -307,6 +307,14 @@ static void SetSys (const char* Sys)
             NewSymbol ("__OSIC1P__", 1);
             break;
 
+        case TGT_CREATIX_EMUL:
+            NewSymbol ("__CREATIX_EMUL__", 1);
+            break;
+            
+        case TGT_CREATIX_NATIVE:
```

Comment:
I think it's difficult to build native and emul libs in one subdir using the current makefile in libsrc. Using two targets is the easiest way. What is your suggestion to solve this problem?